### DISCUSSION TITLE
Listview component (for lack of better naming)

### DISCUSSION THREAD

**Discussion by mgambill** [tailwindlabs/headlessui#294] [Ideas]
The Listview (or whatever we call it)  would be the same as the listbox with the following changes:

1. it doesn't need the open/close functionality
2. left / right keys would trigger up/down actions. since the developer could be creating vertical or horizontal controls
3. multi-select would be nice 

This component would allow for developers to build various components: tabs, accordions, sidebars... etc. 

I've got a working prototype here:  
https://codesandbox.io/s/xenodochial-montalcini-qc1f9?file=/src/components/listbox/listbox.ts

**Comment by madmod** [comment]
This could be used to make a keyboard navigable 2d grid which could be very nice. (Think Netflix UI on tvs)
